data.sql

```diff
@@ -13,4 +13,33 @@ INSERT INTO public.animals(
     INSERT INTO public.animals(
 	name, date_of_birth, escape_attempts, neutered, weight_kg)
 	VALUES ('Devimon', '2017-05-12', 5, '1', '11');
-    
\ No newline at end of file
+    
+	INSERT INTO public.animals(
+	name, date_of_birth, escape_attempts, neutered, weight_kg)
+	VALUES ('Charmander', '2020-02-08', '0', '0', '11');
+
+	INSERT INTO public.animals(
+	name, date_of_birth, escape_attempts, neutered, weight_kg)
+	VALUES ('Plantmon', '2021-11-15', '2', '1', '5.7');
+
+	INSERT INTO public.animals(
+	name, date_of_birth, escape_attempts, neutered, weight_kg)
+	VALUES ('Squirtle', '1993-04-02', '3', '0', '12.13');
+
+	INSERT INTO public.animals(
+	name, date_of_birth, escape_attempts, neutered, weight_kg)
+	VALUES ('Angemon', '2005-06-12', '1', '1', '45');
```

Comment:
- [ ] Please make sure to assign the proper weight for the first 4 animals as specified in the project requirements as they should have a negative prefix.